en-US/snippets/2017/mar2017_b.lang

```diff
@@ -115,26 +115,27 @@ Your browser is ready for a makeover. Don’t worry, you’ve got tons of option
 
 
 # Appears on Mobile Activation screen; For context, refer to https://bug1336204.bmoattachments.org/attachment.cgi?id=8833047
-# This is a button. Be mindful of the character limit.
+# This is a button. Keep the translation short.
 ;Send
 Send
 
 
+# This is a tool tip.
```

Comment:
nit: tooltip, maybe also "This is a tooltip on the UI element to dismiss the widget"?